tools/tap-driver

```diff
@@ -284,9 +284,12 @@ class TapDriver(Driver):
             if code != "SKIP":
                 skipped = False
 
-        if not self.errored and returncode:
-            self.report_error("process failed: %d" % returncode)
-            self.errored = True
+        if not self.errored:
+            if returncode == 77:
+                skipped = True
```

Comment:
So exit code 77 is a magic one that means "skip"? I don't see that in the [TAP specification](http://testanything.org/tap-specification.html), and a [Google search](https://www.google.com/search?q=tap+exit+code+77) isn't that enlightening either. Can you please add a reference/comment for that?

Reply:
I chose 77 as a way for simple executable tests to communicate the fact that they should be skipped. I still see usage of it in src/retest/retest.c ... although it's used a lot less than before.